pkg/distro/rhel9/distro_internal_test.go

```diff
@@ -1,13 +1,68 @@
 package rhel9
 
 import (
+	"fmt"
+	"math/rand"
 	"testing"
 
+	"github.com/osbuild/images/internal/common"
+	"github.com/osbuild/images/pkg/blueprint"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 
 	"github.com/osbuild/images/pkg/distro"
 )
 
+// math/rand is good enough in this case
+/* #nosec G404 */
+var rng = rand.New(rand.NewSource(0))
+
+func TestEC2Partitioning(t *testing.T) {
+	testCases := []struct {
+		distro      string
+		bootSizeMiB int
+	}{
+		// x86_64
```

Comment:
it says `x86_64` here but below it checks both x86_64 and aarch64?

Reply:
Removed. :)